.github/workflows/docker.yml

```diff
@@ -4,6 +4,8 @@ on:
   push:
     tags:
       - 'v*'
+  schedule:
+    - cron: '11 0 * * *'
```

Comment:
This would run at 00:11, right? Any reasoning for this specific time, or just randomly picked?

Reply:
Ooops, should have added a comment. Wanted to see if it would run on my fork at 11am. But it looks like it didn't (and I got the minute vs. hour mixed up too). Will fix. What time should we pick? Just 00:00?

Reply:
I had to check https://crontab.guru for this comment :') 

00:00 works!